lte/gateway/deploy/roles/agw_docker/tasks/main.yml

```diff
@@ -0,0 +1,195 @@
+---
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Include vars of all.yaml
```

Comment:
can as above this is introducing code duplication. can you refactor existing ansible roles and fork deployment for using debian package and docker containers.